usr/local/bin/install-cylc8

```diff
@@ -31,7 +31,7 @@ mamba --version
 
 LABEL="cylc-8"
 conda_env=cylc-full-build
-mamba env remove -n "${conda_env}"
+mamba env remove -n "${conda_env}" 2>/dev/null || true
```

Comment:
`mamba env remove` fails if the environment is missing (this must be a change in behaviour)